### ISSUE TITLE
trying to load libraries in wrong place in Mac

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [TerraME/terrame#989]
terrame is trying to load libraries from wrong places while testing `CellularSpace:save`.

```
Testing save
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/.libs/version.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiaaux/.libs/gg_sqlaux.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiaaux/.libs/gg_utf8.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiaexif/.libs/gaia_exif.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_advanced.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_endian.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_ewkt.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_extras.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_geoJSON.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_geodesic.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_geometries.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_geoscvt.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_gml.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_kml.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_lwgeom.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_relations.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_relations_ext.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_shape.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_transform.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_vanuatu.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_voronoj.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_wkb.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_wkt.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/gaiageo/.libs/gg_xml.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gaia_cvt_gpkg.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkgAddGeometryColumn.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkgBinary.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkgCreateBaseTables.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkgCreateTilesTable.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkgCreateTilesZoomLevel.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkgGetImageType.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkgInsertEpsgSRID.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkgMakePoint.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkg_add_geometry_triggers.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkg_add_spatial_index.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkg_add_tile_triggers.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkg_get_normal_row.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/geopackage/.libs/libgeopackage_la-gpkg_get_normal_zoom.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-extra_tables.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-mbrcache.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-metatables.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-spatialite.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-spatialite_init.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-statistics.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualXL.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualbbox.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualdbf.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualfdo.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualgpkg.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualnetwork.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualshape.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualspatialindex.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/spatialite/.libs/libsplite_la-virtualxpath.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/shapefiles/.libs/shapefiles.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/shapefiles/.libs/validator.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/dxf/.libs/dxf_load_distinct.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/dxf/.libs/dxf_load_mixed.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/dxf/.libs/dxf_loader.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/dxf/.libs/dxf_parser.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/dxf/.libs/dxf_writer.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/md5/.libs/gaia_md5.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/md5/.libs/md5.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_00.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_01.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_02.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_03.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_04.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_05.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_06.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_07.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_08.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_09.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_10.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_11.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_12.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_13.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_14.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_15.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_16.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_17.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_18.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_19.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_20.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_21.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_22.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_23.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_24.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_25.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_26.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_27.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_28.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_29.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_30.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_31.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_32.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_33.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_34.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_35.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_36.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_37.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_38.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_39.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_40.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_41.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_42.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_43.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_44.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_45.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_extra.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_prussian.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_wgs84_00.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-epsg_inlined_wgs84_01.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/srsinit/.libs/libsrsinit_la-srs_init.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/connection_cache/.libs/alloc_cache.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/virtualtext/.libs/virtualtext.gcda: cannot open: No such file or directory
profiling: /Users/avancini/terralib/3rdparty/terralib-3rdparty-macosx-el-capitan/libspatialite-4.2.0/src/wfs/.libs/wfs_in.gcda: cannot open: No such file or directory
Segmentation fault: 11
```

**Comment by avancinirodrigo** [comment]
This directory is being creating in my machine. It seems associated to environment variable of the terralib.

**Comment by pedro-andrade-inpe** [comment]
The new error is:
```
Testing tests/database/alternative/CellularSpace.lua
Testing save
Wrong execution, got error: 'SWIG_SystemError:boost::bad_format_string: format-string is ill-formed'.
```

**Comment by avancinirodrigo** [comment]
Tests in database depends on #1052.

**Comment by avancinirodrigo** [comment]
This database error is not associeted with this issue. Please open a new issue.

**Comment by avancinirodrigo** [comment]
This erros was assicieted to a flag in libspatialite compilation.
`--enable-gcov: turn on code coverage analysis tools`
